_includes/default-header.html

```diff
@@ -60,9 +60,15 @@
           </div>
 
           <div class="navbar-item has-dropdown is-hoverable">
-            <a class="navbar-link" href="{% link code-of-conduct.md %}">Policies</a>
+            <a class="navbar-link">Policies & Procedures</a>
             <div class="navbar-dropdown">
-              <a class="navbar-item" href="{{ site.github.owner_url }}/policies-procedures-and-docs/blob/main/CODE_OF_CONDUCT.md"> Code of Conduct </a>
+              <a class="navbar-item" href="{% link code-of-conduct.md %}"> Code of Conduct </a>
+              <a class="navbar-item" href="{% link openseeds/cohort-procedures-and-templates/cohort-call-facilitator-guide.md %}"> Cohort Call Facilitator Guide </a>
+              <a class="navbar-item" href="{% link openseeds/cohort-procedures-and-templates/expert-guide.md %}"> Expert Guide </a>
+              <a class="navbar-item" href="{% link openseeds/cohort-procedures-and-templates/graduation-guide.md %}"> Graduation Guide </a>
+              <a class="navbar-item" href="{% link openseeds/cohort-procedures-and-templates/microgrants.md %}"> Microgrants </a>
+              <a class="navbar-item" href="{% link openseeds/cohort-procedures-and-templates/speedblog.md %}"> Speedblog </a>
+              <a class="navbar-item" href="{% link openseeds/cohort-procedures-and-templates/video-facilitator-guide.md %}"> Video Facilitator Guide </a>
```

Comment:
I think these links are specific to openseeds. Here you should probably link to the content in `policies-procedures-and-docs` folder

Reply:
I had a thought.
We can have a `Policies & Procedures` dropdown menu in both the _main_ and `/openseeds` navbars.
The difference would be in the content of the menu (ie one would be specific to openseeds).

Let me know if this sounds redundant. please.

Reply:
Good idea